src/components/views/SSISwipeRowViewItem/index.tsx

```diff
@@ -12,15 +12,16 @@ export interface IProps {
   viewItem: JSX.Element
   listIndex: number
   onPress: () => Promise<void>
-  onDelete: () => Promise<void>
+  onDelete?: () => Promise<void>
```

Comment:
add a comment here to revert this optional behavior, WAL-410 as this onDelete should be mandatory

Reply:
Added.